tasks/rh-sign-image-cosign/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Install the CRDs so we can create/get them
+.github/scripts/install_crds.sh
+
+# Add RBAC so that the SA executing the tests can retrieve CRs
+kubectl apply -f .github/resources/crd_rbac.yaml
```

Comment:
I wonder if this is needed here at all. This is typically needed for tasks that create InternalRequest CRs (or some other CRs) and that involve kubectl calls. But this task doesn't do anything like that, so I don't think this is needed here.

Here's an example where this is not needed: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/create-pyxis-image/tests/pre-apply-task-hook.sh

Reply:
Yeah, it's not needed. Removed.

Reply:
ok, thanks.